config/log4j2.properties

```diff
@@ -0,0 +1,36 @@
+status = error
+name = LogstashPropertiesConfig
+
+property.filename = /tmp/logstash/logstash.log
```

Comment:
@fbaligand ES will be moving to `.properties`. I don't think it's happening in the 5.0 timeframe, but it is definitely on the cards to move to `.properties` _and_ to log4j2. It would make sense for LS to be future proof :). 

I am +1 on allowing logf4j xml config for advanced users, but for most `.properties` should suffice.